libvast/CMakeLists.txt

```diff
@@ -8,6 +8,10 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/vast/config.hpp.in
 include_directories(${CMAKE_CURRENT_SOURCE_DIR})
 include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
+# get header files; only needed by CMake generators,
+# e.g., for creating proper Xcode projects
```

Comment:
Would you mind rephrasing this comment to: *Some CMake generators (e.g., XCode) require header files in addition to the sources.*